source/scripts/api_script.js

```diff
@@ -33,6 +33,7 @@ async function fetchRecipes(queries, callbackFn) {
         fetch(url)
             .then((response) => response.json())
             .then(data => {
+                console.log(data.results);

```

Comment:
get rid of the console.log

Reply:
just removed it and any other console logs we didn't need